### PR TITLE
Pull up ce 11012021

### PR DESCRIPTION
**Pull request by tomglvng**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Oro/Bundle/PimDataGridBundle/Datasource/AssociatedProductDatasource.php [hunk 1]
```diff
@@ -26,6 +30,21 @@ class AssociatedProductDatasource extends ProductDatasource
     /** @var string */
     protected $sortOrder;
 
+    // todo merge remove null
+    /** @var NormalizerInterface|null */
+    private $internalApiNormalizer;
```
Comment (StevenVAIDIE) [inline src/Oro/Bundle/PimDataGridBundle/Datasource/AssociatedProductDatasource.php:35]: You should make internalApiNormalizer required (remove all todo)

File: src/Oro/Bundle/PimDataGridBundle/Datasource/AssociatedProductModelDatasource.php [hunk 2]
```diff
@@ -26,6 +30,21 @@ class AssociatedProductModelDatasource extends ProductDatasource
     /** @var string */
     protected $sortOrder;
 
+    // todo merge remove null
+    /** @var NormalizerInterface|null */
```
Comment (StevenVAIDIE) [inline src/Oro/Bundle/PimDataGridBundle/Datasource/AssociatedProductModelDatasource.php:34]: Same here

**Review by StevenVAIDIE — APPROVED** [review]
GTM after fix

**Review by willy-ahva — APPROVED** [review]

**Review by mr-prud — APPROVED** [review]
Good Job !

**Pull request merged**